File: Golang/08_geyser_bath_capacity_beginner.go
package main

import "fmt"

func main() {
	var T int
	fmt.Scan(&T)
	for i := 0; i < T; i++ {
		var X, Y int
		fmt.Scan(&X, &Y)
		maxPeople := X / (2 * Y)
		fmt.Println(maxPeople)
	}
}

/**********************************************************
 * @INFO
 * Code by Ashish Singh
 * @INFO
 * Gmail - ashishlodhi5559@gmail.com
 * @INFO
 * Github - AshishBytes
 * @INFO
 * LinkedIn - ashishbytes
 * @INFO
 * Instagram - @itz_ash._u
 * @INFO
 * Twitter - @itz_ash_u
 * @INFO
 **********************************************************/

File: C++/08_geyser_bath_capacity_beginner.cpp
#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    while (T--) {
        int X, Y;
        cin >> X >> Y;
        int maxPeople = X / (2 * Y);
        cout << maxPeople << endl;
    }
    return 0;
}
